pages/common/pio-remote.md

```diff
@@ -0,0 +1,33 @@
+# pio remote
+
+> Helper command for PlatformIO Remote Development.
+> `pio remote command` usually takes the same arguments as its counterpart `pio command`.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/remote/index.html>.
+
+- List all active Remote Agents:
+
+`pio remote agent list`
+
+- Start a new Remote Agent with a specific name and share it with friends:
+
+`pio remote agent start --name {{agent_name}} --share {{friend@mail.org}}`
```

Comment:
I can see a single friend. 😇 
Adapt the explanation? ("with a friend")? Or show how to pass two mail addresses as parameters?

Reply:
I just didn't know how to specify multiple email adresses, since it's not in the docs. But I found in the [forum](https://community.platformio.org/t/unsure-how-to-use-pio-remote-share/19045/3), so I'll chnage the example.